dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,56 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Escort a Dvaered colonel to a system">
+ <unique />
+ <priority>1</priority>
```

Comment:
Priority 1 should be reserved for more critical missions. In this case, either default or something like 4 would make sense.

Reply:
Ah, I had been under the impression that 1 made it least priority.